gemini/__init__.py

```
@@ -0,0 +1,5 @@
+"""
+Gemini CLI拡張フレームワークパッケージ
+"""
+
+from gemini.supergemini import *
```

Ruff (0.12.2)

`from gemini.supergemini import *` used; unable to detect undefined names

(F403)